src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -84,6 +84,57 @@
 ['\"]        # Single or double quote.
 """, re.VERBOSE)
 
+FTI_BEHAVIOR_PATTERN = re.compile(r"""
+\s           # Whitespace.
+<element     #
+\s+          #
+value=       #
+['\"]        # Single or double quote.
+(?P<import>  # Start of 'import' variable.
+\S+          # Non-whitespace string.
+)            # End of 'import' variable.
+['\"]        # Single or double quote.
+""", re.VERBOSE)
+
+FTI_SCHEMA_PATTERN = re.compile(r"""
+\s           # Whitespace.
+name=        #
+['\"]        # Single or double quote.
+schema       #
+['\"]        # Single or double quote.
+>            # End of opening tag
+(?P<import>  # Start of 'import' variable.
+\S+          # Non-whitespace string.
```

Comment:
I personally prefer to use explicit lookaheads in such situations:
`[^<]+` would match all characters but not a `<`; `\S` would match `<` too, which makes it less specific: if the expression is too greed we would probably have too much in the `import` group.

For instance when there is a typo (two `<` characters), it wouldn't match correctly:
```python
>>> import re
>>> re.search(r'>(\S+)<', '<foo>bar</foo>').group(1)
'bar'
>>> re.search(r'>(\S+)<', '<foo>bar<</foo>').group(1)
'bar<'
```

although one could argument that then the XML would be invalid and therefore it is okay when the result is wrong.
Just my thoughts :wink: